screens/AirQualityComponent.js

```diff
@@ -0,0 +1,117 @@
+import { View, Text, ScrollView, StyleSheet, FlatList } from "react-native";
+import React, { useEffect, useState } from "react";
+import GasInfo from "../components/GasInfo";
+
+const AirQualityComponent = ({ weatherDetails }) => {
+  const [airQualityConditionText, setAirQualityConditionText] = useState("");
+
+  const handleAirQualityText = () => {
+    const aqi = weatherDetails?.current?.air_quality?.["us-epa-index"];
+    switch (aqi) {
+      case 1:
+        return "Good";
+      case 2:
+        return "Moderate";
+      case 3:
+        return "Sensitive";
+      case 4:
+        return "Unhealthy";
+      case 5:
+        return "Hazardous";
+    }
+  };
+  useEffect(() => {
+    setAirQualityConditionText(handleAirQualityText());
+  }, []);
+
+  const gases = ["co", "no2", "o3", "so2"];
+  const gasArr = gases.map((gas) => {
+    return { name: gas, qty: weatherDetails?.current?.air_quality[gas] };
+  });
```

Comment:
```suggestion
  const gasArr = gases.map((gas) => ({ name: gas, qty: weatherDetails?.current?.air_quality[gas] });
```

this also works? instead of the extra return. nitpicking :) 
dont commit from here, do it from local where linter and all will check code.